assets/source/setup-guide/app/views/WizardApp.js

```diff
@@ -70,11 +82,13 @@ const WizardApp = () => {
 				setIsBusinessConnected,
 				isBusinessConnected,
 			},
+			isClickable: isBusinessConnected,
 		},
 		{
 			key: 'claim-website',
 			container: ClaimWebsite,
 			label: __( 'Claim your website', 'pinterest-for-woocommerce' ),
+			isClickable: isBusinessConnected || isDomainVerified,
```

Comment:
I am not sure how you could get a verified domain without connecting your account? When I see the code in isDomainVerified hook

```		
export const isDomainVerified = ( state ) => {
	if ( undefined === state?.settings?.account_data ) {
		return;
	}
```
I see that is returning undefined if the account_data is undefined. If I try to reproduce it I get the following response from the API:

![image](https://user-images.githubusercontent.com/2488994/149350433-486d1d94-72ed-414a-9668-9b6dd51c5288.png)

`account_data` is null therefore isDomainVerified does not return `true`. 

In case a user can have this situation, I updated the code see: [9760a4c.](https://github.com/woocommerce/pinterest-for-woocommerce/pull/318/commits/9760a4ce5078a5c874b86efa8216668aa72f710f)

Reply:
> I am not sure how you could get a verified domain without connecting your account?

General operations on UI won't get into this case.

Sorry 🙏  I should have added these notes. I made some changes locally to show that the order of dependence between steps should be a more appropriate condition in step 3, and the `isDomainVerified` should be logically unneeded in step 2.